app/models/match_prioritization/match_group_disability.rb

```diff
@@ -0,0 +1,30 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+module MatchPrioritization
+  class MatchGroupDisability < Base
+    def self.title
+      'Match Group with Disability'
+    end
+
+    def self.prioritization_for_clients(scope, match_route:) # rubocop:disable Lint/UnusedMethodArgument
+      # NOTE: after Rails 6.1 upgrade, these could be written c_t[:veteran].desc.nulls_last
+      match_group = Arel.sql(c_t[:match_group].asc.to_sql + ' NULLS LAST') # 1, 2, 3
+      chronic = Arel.sql(c_t[:chronic_homeless].desc.to_sql + ' NULLS LAST') # chronic prioritized over non-chronic
+      entry_date = Arel.sql(c_t[:entry_date].asc.to_sql + ' NULLS LAST') # older prioritized over newer
+      vispdat_score = Arel.sql(c_t[:vispdat_score].desc.to_sql + ' NULLS LAST') # higher prioritized over lover
+
+      # TODO: check disability_verified_on too?
+      has_disability = c_t[:physical_disability].eq(true).or(c_t[:developmental_disability].eq(true))
```

Comment:
which fields should be checked for determining disability?

Reply:
@awisema you mentioned we should use a rule for this instead. It looks like we already have 3 rules for disability: DevelopmentalDisability, PhysicalDisablingCondition, VerifiedDisability. Can we use one/several of those to achieve the same result? I'm not sure how those get applied